content/en/getting_started/synthetics/browser_test.md

```diff
@@ -19,77 +22,117 @@ further_reading:
-        ```
+```text
+An alert is triggered if your test fails for 0 minutes from any 3 of 13 locations
+```
 
-         **Note**: By default, there is a 300ms wait before retrying a test that failed. This interval can be configured via the [API][3].
+#### Notify your team
 
-    - **Notify**: Write an alert message and specify which email addresses should be notified when the alert is triggered. No additional set up is required to start receiving alert emails from Datadog. You can also use [integrations][4], such as Slack, PagerDuty, webhooks, etc., to receive alert notifications.
-    - Click **Save & Edit Recording**.
+Add an alert name to the **Monitor Name** field and write a message for the alert. You can tag other teams, specify which service and team members receive the alert notifications, and use [integrations][4] such as Slack, PagerDuty, and webhooks.
+
+You can set time for your alert notification to re-notify if the alert has not been resolved and define the priority of the alert, ranging from **P5 (Info)** to **P1 (Critical)**.
+
+When you're ready to run your test, click **Save Details & Record Test**.
 
 {{< img src="getting_started/synthetics/configured-browser-test.gif" alt="Configured browser test"  style="width:90%;">}}
 
-## Record your test steps
+### Create recording
+
+Once your test configuration has saved, Datadog prompts you to download and install the [Datadog test recorder][5] extension. You can run browser tests on Google Chrome, Microsoft Edge, and Firefox. 
 
-Once the test configuration is saved, Datadog will prompt you to download the [Datadog test recorder][5] extension. Browser tests can be only recorded on **[Google Chrome][6]**. Download and install the extension.
+To begin recording your test steps, click **Start Recording**. 
 
-Once this extension is installed, begin recording your test steps by clicking the **Start Recording** button. Navigate your page in the iframe to the right of the recording options. When you select a div, image, or any area of your page, the steps are recorded and used to create steps within the browser test. Learn more about each step in the [browser test steps doc][7].
+You can navigate through your page in the iframe to the right of the recording button. By selecting a div, image, or any area of your page, Datadog records and creates your steps in the browser test. For more information, see [Browser Test Steps][7].
 
-For example, to record test steps that map a user's journey from adding an item to cart to successful checkout:
+To end recording your test steps, click **Stop Recording**.
 
-1. Navigate to one of the furniture sections, for instance **Chairs**, and select **Add to cart**.
-2. Click on **Cart**, click **Checkout**.
-3. Manually add the **Assertion** “Test text is present on the active page” to confirm the words “Thank you” are on the page.
+The example below demonstrates how to map a user journey from adding an item to a cart to successfully checking out:
```

Comment:
Might be helpful to provide the link to the Shopist website here? Is that the example website we're talking about?

Reply:
```suggestion
The example below demonstrates how to map a user journey from adding an item to a cart to successfully checking out in `https://www.shopist.io`:
```